projects/netgrif-components-core/src/lib/task-content/model/task-fields.ts

```diff
@@ -0,0 +1,8 @@
+import {DataField} from '../../data-fields/models/abstract-data-field';
+
+export interface TaskFields {
```

Comment:
This interface is now exported in public-api file for task-content module.
Please include it in exported objects.
https://github.com/netgrif/components/blob/NAE-1577_2/projects/netgrif-components-core/src/lib/task-content/public-api.ts

Reply:
Corrected